src/blocks/server.blocks/middleware/__error/middleware__error.node.js

```diff
@@ -53,23 +52,39 @@ modules.define('middleware__error', ['config', 'logger', 'util'], function (prov
 
     /**
      * Log error message and set statusCode to response
+     * If the error code does not have its own template
+     * show 500 page and logging this error in console
      * @param err - {Error} error
-     * @param req - {Object} request object
      * @param res - {Object} response object
      */
-    function preparation(err, req, res) {
-        var code = err.code || 500,
-            terr = terror.ensureError(err);
+    function preparation(err, res) {
+        var supportCodes = [404, 500],
```

Comment:
No. My code much better, it more comprehensive and extensible. If in future we will want to add new error template, we must add one item to array `supportCodes ` and it`s all.